trompace/mutations/audioobject.py

```diff
@@ -140,3 +140,60 @@ def mutation_delete_audio_object(identifier: str):
     """
 
     return format_mutation("DeleteAudioObject", {"identifier": identifier})
+
+
+def mutation_remove_audio_object_work_example(audio_object_identifier: str, work_identifier: str):
+    """Returns a mutation for creating removing a audio object as an example of a work.
+
+    Arguments:
+        audio_object_identifier: The unique identifier of the audio object.
+        work_identifier: The unique identifier of the work that the audio object is an example of.
+
+    Returns:
+        The string for the mutation for removing a audio object as an example of the work.
+    """
+
+    return format_link_mutation("RemoveAudioObjectExampleOfWork", audio_object_identifier, work_identifier)
+
+
+def mutation_merge_audio_object_encoding(audio_object_identifier_1: str, audio_object_identifier_2: str):
```

Comment:
Can you have a look at the documentation that I added for mediaobject and copy it more or less here?
https://github.com/trompamusic/trompa-ce-client/blob/6bc74c45422a1485459d89d3cdc7001bb36d01ae/trompace/mutations/mediaobject.py#L172

Note also that I called it `mediaobject` not `media_object`, because we're using _ to separate words in the method. Everywhere else I try to spell CE nodes with no _. Can you also look at the other methods in this file?